homeassistant/components/satel_integra/__init__.py

```diff
@@ -1,14 +1,17 @@
 """Support for Satel Integra devices."""
+import collections
 import logging
 
 import voluptuous as vol
 
-from homeassistant.const import EVENT_HOMEASSISTANT_STOP, CONF_HOST
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP, CONF_HOST, CONF_PORT
 from homeassistant.core import callback
 from homeassistant.helpers import config_validation as cv
 from homeassistant.helpers.discovery import async_load_platform
 from homeassistant.helpers.dispatcher import async_dispatcher_send
 
+REQUIREMENTS = ['satel_integra==0.3.4']
```

Comment:
We have removed this. Please remove it. It's now enough to define the requirements in the manifest.json file and run the scripts to validate the manifest.

Reply:
Done - this was a merge error on my behalf.